tiledb/libtiledb.pyx

```diff
@@ -1659,6 +1660,25 @@ cdef ArrayPtr preload_array(uri, mode, key, timestamp, ctx=None):
     rc = tiledb_array_alloc(ctx_ptr, uri_ptr, &array_ptr)
     if rc != TILEDB_OK:
         _raise_ctx_err(ctx_ptr, rc)
+    
+    cdef tiledb_config_t* config_ptr = NULL
+    cdef tiledb_error_t* err_ptr = NULL
+    if key is not None:
+        rc = tiledb_config_alloc(&config_ptr, &err_ptr)
```

Comment:
This context is leaking, we need to free it after `tiledb_array_set_config`. Will push a fix in a sec.

Reply:
Thank you for catching and fixing that.